opm/test_util/EclFilesComparator.cpp

```diff
@@ -305,3 +217,402 @@ double ECLFilesComparator::average(const std::vector<double>& vec) {
     double sum = std::accumulate(vec.begin(), vec.end(), 0.0);
     return sum/vec.size();
 }
+
+
+
+void RegressionTest::printResultsForKeyword(const std::string& keyword) const {
+    std::cout << "Deviation results for keyword " << keyword << " of type "
+        << ecl_util_get_type_name(ecl_file_iget_named_type(ecl_file1, keyword.c_str(), 0))
+        << ":\n";
+    const double absDeviationAverage = average(absDeviation);
+    const double relDeviationAverage = average(relDeviation);
+    std::cout << "Average absolute deviation = " << absDeviationAverage  << std::endl;
+    std::cout << "Median absolute deviation  = " << median(absDeviation) << std::endl;
+    std::cout << "Average relative deviation = " << relDeviationAverage  << std::endl;
+    std::cout << "Median relative deviation  = " << median(relDeviation) << "\n\n";
+}
+
+
+
+void RegressionTest::boolComparisonForOccurrence(const std::string& keyword, int occurrence1, int occurrence2) const {
```

Comment:
i don't see how i can do the equivalent using ecl_kw_equal. it won't tell me where it differs.